<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>ConfigDialog</class>
	<widget class="QDialog" name="ConfigDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>600</width>
				<height>680</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>çº¢è–¯æ™ºæ’­ - æ’ä»¶é…ç½®</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				/* é—®å·å›¾æ ‡æ ·å¼ */
				QLabel#lblHelpTime, QLabel#lblHelpNoise, QLabel#lblHelpHistory, QLabel#lblHelpChain, QLabel#lblHelpDuck {
				background-color: #444;
				color: #ccc;
				border-radius: 9px;
				padding: 0px;
				font-weight: bold;
				font-size: 11px;
				min-width: 18px;
				max-width: 18px;
				min-height: 18px;
				max-height: 18px;
				qproperty-alignment: AlignCenter;
				}
				QLabel#lblHelpTime:hover, QLabel#lblHelpNoise:hover, QLabel#lblHelpHistory:hover, QLabel#lblHelpChain:hover, QLabel#lblHelpDuck:hover {
				background-color: #DF6A46;
				color: white;
				cursor: help;
				}

				/* ğŸ¯ æ ¸å¿ƒä¿®å¤ï¼šå…¨å±€ç»Ÿä¸€é«˜åº¦ä¸º 28px */
				QSpinBox, QLineEdit, QComboBox, QPushButton {
				min-height: 28px;
				max-height: 28px;
				font-size: 12px;
				margin: 4px;
				}

				/* æ ‡ç­¾åˆ—è¡¨å®¹å™¨æ ·å¼ */
				QListWidget#listDuckTags {
				background: transparent;
				border: 1px solid #3a3a3e;
				border-radius: 4px;
				outline: none;
				}
				QListWidget#listDuckTags::item {
				border: none;
				}
			</string>
		</property>
		<layout class="QVBoxLayout" name="mainLayout">
			<property name="spacing">
				<number>12</number>
			</property>
			<property name="leftMargin">
				<number>24</number>
			</property>
			<property name="topMargin">
				<number>24</number>
			</property>
			<property name="rightMargin">
				<number>24</number>
			</property>
			<property name="bottomMargin">
				<number>24</number>
			</property>
			<item>
				<layout class="QGridLayout" name="gridLayout">
					<property name="verticalSpacing">
						<number>12</number>
					</property>
					<property name="horizontalSpacing">
						<number>12</number>
					</property>
					<item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label">
							<property name="text">
								<string>æ’æ’­åª’ä½“æº:</string>
							</property>
						</widget>
					</item>
					<item row="0" column="2">
						<widget class="QComboBox" name="comboMediaSource"/>
					</item>
					<item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label_2">
							<property name="text">
								<string>è¯­éŸ³åŒ…æ ¹ç›®å½•:</string>
							</property>
						</widget>
					</item>
					<item row="1" column="2">
						<layout class="QHBoxLayout" name="pathLayout">
							<property name="spacing">
								<number>8</number>
							</property>
							<item>
								<widget class="QLineEdit" name="editVoicePath"/>
							</item>
							<item>
								<widget class="QPushButton" name="btnBrowse">
									<property name="text">
										<string>æµè§ˆ...</string>
									</property>
								</widget>
							</item>
						</layout>
					</item>
					<item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label_3">
							<property name="text">
								<string>æŠ¥æ—¶é—´éš”(ç§’):</string>
							</property>
						</widget>
					</item>
					<item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
						<widget class="QLabel" name="lblHelpTime">
							<property name="text">
								<string>?</string>
							</property>
							<property name="toolTip">
								<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;background-color:#222; color:#eee; padding:5px;&quot;&gt;è®¾ç½®è‡ªåŠ¨æŠ¥æ—¶çš„è§¦å‘é¢‘ç‡ã€‚&lt;br/&gt;ç³»ç»Ÿå°†åœ¨è®¾å®šçš„æœ€å°å’Œæœ€å¤§ç§’æ•°ä¹‹é—´&lt;br/&gt;éšæœºé€‰æ‹©ä¸€ä¸ªæ—¶é—´ç‚¹è¿›è¡ŒæŠ¥æ—¶ã€‚&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
							</property>
						</widget>
					</item>
					<item row="2" column="2">
						<layout class="QHBoxLayout" name="timeLayout">
							<property name="spacing">
								<number>8</number>
							</property>
							<item>
								<widget class="QSpinBox" name="spinTimeMin">
									<property name="minimumSize">
										<size>
											<width>100</width>
											<height>0</height>
										</size>
									</property>
									<property name="maximum">
										<number>9999</number>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_4">
									<property name="text">
										<string>è‡³</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QSpinBox" name="spinTimeMax">
									<property name="minimumSize">
										<size>
											<width>100</width>
											<height>0</height>
										</size>
									</property>
									<property name="maximum">
										<number>9999</number>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="horizontalSpacer">
									<property name="orientation">
										<enum>Qt::Horizontal</enum>
									</property>
									<property name="sizeHint" stdset="0">
										<size>
											<width>40</width>
											<height>20</height>
										</size>
									</property>
								</spacer>
							</item>
						</layout>
					</item>
					<item row="3" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label_5">
							<property name="text">
								<string>æ··æ·†é—´éš”(ç§’):</string>
							</property>
						</widget>
					</item>
					<item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
						<widget class="QLabel" name="lblHelpNoise">
							<property name="text">
								<string>?</string>
							</property>
							<property name="toolTip">
								<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;background-color:#222; color:#eee; padding:5px;&quot;&gt;è®¾ç½®æ’­æ”¾æ··æ·†éŸ³ï¼ˆå¦‚é”®ç›˜å£°ã€é¼ æ ‡å£°ï¼‰çš„é¢‘ç‡ã€‚&lt;br/&gt;ç³»ç»Ÿå°†åœ¨è®¾å®šçš„æœ€å°å’Œæœ€å¤§ç§’æ•°ä¹‹é—´éšæœºè§¦å‘ï¼Œ&lt;br/&gt;å¢åŠ ç›´æ’­é—´çš„çœŸå®æ„Ÿã€‚&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
							</property>
						</widget>
					</item>
					<item row="3" column="2">
						<layout class="QHBoxLayout" name="noiseLayout">
							<property name="spacing">
								<number>8</number>
							</property>
							<item>
								<widget class="QSpinBox" name="spinNoiseMin">
									<property name="minimumSize">
										<size>
											<width>100</width>
											<height>0</height>
										</size>
									</property>
									<property name="maximum">
										<number>9999</number>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_6">
									<property name="text">
										<string>è‡³</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QSpinBox" name="spinNoiseMax">
									<property name="minimumSize">
										<size>
											<width>100</width>
											<height>0</height>
										</size>
									</property>
									<property name="maximum">
										<number>9999</number>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="horizontalSpacer_2">
									<property name="orientation">
										<enum>Qt::Horizontal</enum>
									</property>
									<property name="sizeHint" stdset="0">
										<size>
											<width>40</width>
											<height>20</height>
										</size>
									</property>
								</spacer>
							</item>
						</layout>
					</item>
					<item row="4" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label_9">
							<property name="text">
								<string>æ··æ·†å»é‡è½®æ•°:</string>
							</property>
						</widget>
					</item>
					<item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
						<widget class="QLabel" name="lblHelpHistory">
							<property name="text">
								<string>?</string>
							</property>
							<property name="toolTip">
								<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;background-color:#222; color:#eee; padding:5px;&quot;&gt;è®¾ç½®ç³»ç»Ÿè®°å¿†æœ€è¿‘æ’­æ”¾è¿‡çš„æ··æ·†éŸ³æ•°é‡ã€‚&lt;br/&gt;ä¾‹å¦‚è®¾ä¸º30ï¼Œåˆ™ç³»ç»Ÿä¼šå°½é‡é¿å…æ’­æ”¾æœ€è¿‘30æ¬¡&lt;br/&gt;å†…æ’­æ”¾è¿‡çš„æ–‡ä»¶ï¼Œé˜²æ­¢å£°éŸ³é‡å¤å•è°ƒã€‚&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
							</property>
						</widget>
					</item>
					<item row="4" column="2">
						<widget class="QSpinBox" name="spinHistorySize">
							<property name="minimumSize">
								<size>
									<width>100</width>
									<height>0</height>
								</size>
							</property>
							<property name="maximum">
								<number>100</number>
							</property>
						</widget>
					</item>
					<item row="5" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
						<widget class="QLabel" name="label_10">
							<property name="text">
								<string>æ··æ·†è¿æ’­é˜ˆå€¼(ç§’):</string>
							</property>
						</widget>
					</item>
					<item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
						<widget class="QLabel" name="lblHelpChain">
							<property name="text">
								<string>?</string>
							</property>
							<property name="toolTip">
								<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;background-color:#222; color:#eee; padding:5px;&quot;&gt;å¦‚æœéšæœºé€‰ä¸­çš„éŸ³é¢‘æ—¶é•¿å°äºæ­¤é˜ˆå€¼ï¼ˆä¾‹å¦‚6ç§’ï¼‰ï¼Œ&lt;br/&gt;ç³»ç»Ÿä¼šè‡ªåŠ¨è¿½åŠ æ’­æ”¾ç¬¬2ä¸ªç”šè‡³ç¬¬3ä¸ªéŸ³é¢‘ï¼Œ&lt;br/&gt;æ¨¡æ‹Ÿè¿ç»­çš„ç¢ç¢å¿µæ•ˆæœã€‚&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
							</property>
						</widget>
					</item>
					<item row="5" column="2">
						<widget class="QSpinBox" name="spinShortThreshold">
							<property name="minimumSize">
								<size>
									<width>100</width>
									<height>0</height>
								</size>
							</property>
							<property name="maximum">
								<number>60</number>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item>
				<widget class="QGroupBox" name="groupDucking">
					<property name="title">
						<string>éŸ³é¢‘é—ªé¿è®¾ç½®</string>
					</property>
					<layout class="QVBoxLayout" name="duckVLayout">
						<property name="spacing">
							<number>12</number>
						</property>
						<property name="leftMargin">
							<number>16</number>
						</property>
						<property name="topMargin">
							<number>16</number>
						</property>
						<property name="rightMargin">
							<number>16</number>
						</property>
						<property name="bottomMargin">
							<number>16</number>
						</property>
						<item>
							<layout class="QHBoxLayout" name="duckHeaderLayout">
								<item>
									<widget class="QLabel" name="label_11">
										<property name="text">
											<string>é€‰æ‹©éœ€è¦å‹ä½éŸ³é‡çš„èƒŒæ™¯æº:</string>
										</property>
									</widget>
								</item>
								<item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
									<widget class="QLabel" name="lblHelpDuck">
										<property name="text">
											<string>?</string>
										</property>
										<property name="toolTip">
											<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;background-color:#222; color:#eee; padding:5px;&quot;&gt;å½“æ’ä»¶æ’­æ”¾è¯­éŸ³ï¼ˆæŠ¥æ—¶ã€å›å¤ã€æ··æ·†ï¼‰æ—¶ï¼Œ&lt;br/&gt;å‹¾é€‰çš„è¿™äº›OBSéŸ³æºï¼ˆå¦‚èƒŒæ™¯éŸ³ä¹ï¼‰çš„éŸ³é‡ä¼šè‡ªåŠ¨é™ä½ï¼Œ&lt;br/&gt;æ’­æ”¾ç»“æŸåè‡ªåŠ¨æ¢å¤ï¼Œç¡®ä¿è¯­éŸ³æ¸…æ™°ã€‚&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
										</property>
									</widget>
								</item>
								<item>
									<spacer name="horizontalSpacer_3">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>
							</layout>
						</item>
						<item>
							<widget class="QComboBox" name="comboAddDuckSource">
								<property name="placeholderText">
									<string>ç‚¹å‡»æ­¤å¤„é€‰æ‹©è¦æ·»åŠ çš„éŸ³é¢‘æº...</string>
								</property>
							</widget>
						</item>
						<item>
							<widget class="QListWidget" name="listDuckTags">
								<property name="minimumSize">
									<size>
										<width>0</width>
										<height>80</height>
									</size>
								</property>
								<property name="verticalScrollBarPolicy">
									<enum>Qt::ScrollBarAsNeeded</enum>
								</property>
								<property name="horizontalScrollBarPolicy">
									<enum>Qt::ScrollBarAlwaysOff</enum>
								</property>
								<property name="showDropIndicator" stdset="0">
									<bool>false</bool>
								</property>
								<property name="dragEnabled">
									<bool>false</bool>
								</property>
								<property name="selectionMode">
									<enum>QAbstractItemView::NoSelection</enum>
								</property>
								<property name="flow">
									<enum>QListView::LeftToRight</enum>
								</property>
								<property name="isWrapping" stdset="0">
									<bool>true</bool>
								</property>
								<property name="resizeMode">
									<enum>QListView::Adjust</enum>
								</property>
								<property name="viewMode">
									<enum>QListView::IconMode</enum>
								</property>
							</widget>
						</item>
						<item>
							<layout class="QHBoxLayout" name="volLayout">
								<item>
									<widget class="QLabel" name="label_8">
										<property name="text">
											<string>å‹ä½è‡³éŸ³é‡:</string>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QSlider" name="sliderDuckVol">
										<property name="maximum">
											<number>100</number>
										</property>
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QLabel" name="lblVolVal">
										<property name="text">
											<string>20%</string>
										</property>
									</widget>
								</item>
							</layout>
						</item>
					</layout>
				</widget>
			</item>
			<item>
				<widget class="QDialogButtonBox" name="buttonBox">
					<property name="standardButtons">
						<set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>